sentinel-adapter/sentinel-spring-webmvc-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/AbstractSentinelInterceptor.java

```diff
@@ -168,12 +170,21 @@ protected void removeEntryInRequest(HttpServletRequest request) {
     }
 
     protected void traceExceptionAndExit(Entry entry, Exception ex) {
-        if (entry != null) {
-            if (ex != null) {
-                Tracer.traceEntry(ex, entry);
+        if (entry == null) {
+            return;
+        }
+        if (ex == null) {
+            HttpServletRequest httpServletRequest = getHttpServletRequest();
+            if (httpServletRequest == null) {
+                return;
             }
-            entry.exit();
+            ex = (Exception) httpServletRequest.getAttribute(BaseWebMvcConfig.REQUEST_REF_EXCEPTION_NAME);
```

Comment:
For servlet requests, if there is no configuration for the globalExceptionHandler, when a request fails, it will be forwarded to /error. A failed request will be counted as two errors. It is recommended to remove it from the request.attributes after the first handling is completed.

Reply:
Maybe you can run a test?

```
   /**
     * @param request
     * @param rcKey
     * @param step
     * @return reference count after increasing (initial value as zero to be increased) 
     */
    private Integer increaseReferece(HttpServletRequest request, String rcKey, int step) {
        Object obj = request.getAttribute(rcKey);
        
        if (obj == null) {
            // initial
            obj = Integer.valueOf(0);
        }
        
        Integer newRc = (Integer)obj + step;
        request.setAttribute(rcKey, newRc);
        return newRc;
    }
```

Reply:
I contacted you through email. Can we communicate in more detail?

Reply:
Thanks for your advice and I have fix it.